styles/style.css

```diff
@@ -0,0 +1,174 @@
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: stretch;
+}
+
+.nav-list-parent {
+  display: flex;
+}
+
+.nav-list-parent > .nav-list-child {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 105.633px;
+  height: 36px;
+  background: linear-gradient(rgba(0, 0, 0, 0.2) 0%, rgba(0, 0, 0, 0) 5%, rgba(0, 0, 0, 0) 97%, rgba(0, 0, 0, 0.45) 100%), linear-gradient(rgba(0, 0, 0, 0.2) 0%, rgba(0, 0, 0, 0) 0.2%, rgba(0, 0, 0, 0) 99.8%, rgba(0, 0, 0, 0.2) 100%), linear-gradient(rgb(102, 102, 102) 0%, rgb(94, 94, 94) 50%, rgb(112, 112, 112) 51%, rgb(128, 128, 128) 100%);
+  border: 0.1px solid grey;
+  border-radius: 4px 0 0 4px;
+  list-style: none;
+}
+
+.nav-list-child-input {
+  width: 150px;
+}
+
+.nav-list-parent > .nav-list-child:hover {
+  background: linear-gradient(rgba(0, 0, 0, 0.2) 0%, rgba(0, 0, 0, 0) 5%, rgba(0, 0, 0, 0.616) 97%, rgba(0, 0, 0, 0.45) 100%), linear-gradient(rgba(0, 0, 0, 0.2) 0%, rgba(0, 0, 0, 0) 0.2%, rgba(0, 0, 0, 0) 99.8%, rgba(0, 0, 0, 0.2) 100%), linear-gradient(rgb(102, 102, 102) 0%, rgb(94, 94, 94) 50%, rgb(112, 112, 112) 51%, rgb(128, 128, 128) 100%);
+}
+
+.nav-list-parent > .nav-list-child-input:hover {
+  background: linear-gradient(rgba(0, 0, 0, 0.2) 0%, rgba(0, 0, 0, 0) 5%, rgba(0, 0, 0, 0) 97%, rgba(0, 0, 0, 0.45) 100%), linear-gradient(rgba(0, 0, 0, 0.2) 0%, rgba(0, 0, 0, 0) 0.2%, rgba(0, 0, 0, 0) 99.8%, rgba(0, 0, 0, 0.2) 100%), linear-gradient(rgb(102, 102, 102) 0%, rgb(94, 94, 94) 50%, rgb(112, 112, 112) 51%, rgb(128, 128, 128) 100%);
+}
+
+.nav-list-child > .list-child-text {
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  font-size: 12px;
+  text-align: center;
+}
+
+.list-child-icon {
+  width: 20px;
+  height: 20px;
+}
+
+.list-input-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 12px;
+  width: 100px;
+  border: 1px white inset;
+}
+
+.list-input-form > input {
+  width: 60px;
+  height: 20px;
+  border: none;
+  background: linear-gradient(180deg, rgb(102, 102, 102) 50%, #a3a2a2 100%);
+}
+
+.flex-main-link-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.main-link {
+  text-align: center;
+  text-decoration: none;
+}
+
+.main-text,
+.main-text-bigger {
+  font-weight: bold;
+}
+
+.main-text {
+  font-size: 20px;
+}
+
+.main-text-bigger {
+  font-size: 28px;
+}
+
+.main-title {
+  font-size: 64px;
+}
+
+.flex-paragraph-container {
+  height: 95%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.flex-link-and-icon-container {
+  width: 105%;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.main-link-decoration:hover {
+  text-decoration: underline;
+}
+
+.flex-link-and-icon-container > .main-icon {
+  width: 30px;
+  height: 30px;
+}
+
+.flex-images-section-container {
+  width: 100vh;
```

Comment:
This should be `vw` right? Kindly refer to [this article](http://www.tipue.com/blog/css3-vw-vh/) for clarity.

Also, the overall width of the images all together should align with the width of the background image above. 

![width](https://user-images.githubusercontent.com/31304943/68794207-dfca1500-064e-11ea-8d18-b8443e0a4d8d.png)